src/main/java/com/faforever/api/clan/ClansController.java

```diff
@@ -60,14 +58,13 @@ public MeResult me(Authentication authentication) {
     @ApiResponse(code = 400, message = "Bad Request")})
   @PostMapping(path = "/create", produces = APPLICATION_JSON_VALUE)
   @PreAuthorize("hasRole('ROLE_USER')")
+  @Deprecated // use POST /data/clans instead (with a founder in relationships)
```

Comment:
Can u not pass that text via an argument to the annotation

Reply:
No. Kotlin has this, but Java doesn't